src/gsi/hybrid_ensemble_parameters.f90

```diff
@@ -393,9 +387,9 @@ module hybrid_ensemble_parameters
   integer(i_kind) :: ntotensgrp=1
   integer(i_kind) :: naensloc=1
   integer(i_kind) :: ngvarloc=1
-  integer(i_kind) :: i_ensloccov4tim=0
-  integer(i_kind) :: i_ensloccov4var=0
-  integer(i_kind) :: i_ensloccov4scl=0
+  real(r_kind) :: r_ensloccov4tim
```

Comment:
@shoyokota , why are those options changed to real numbers?  When they are supposed to be used as real numbers in some situations ,  we can always use real function to convert integer to real version.  But their basic usages are used as options with many equality evaluations.  While equality for real numbers are risky or at least not be encouraged to be used.   So, any compelling reasons for this change ?